### DISCUSSION TITLE
DAI 2023-2024 - Practical work 2 - Lucas Lattion and Romain Humair

### DISCUSSION THREAD

**Discussion by HumairRomain** [heig-vd-dai-course/heig-vd-dai-course#137] [Show and tell]
Jeu du pendu

Coté serveur :
- Gérez la logique du jeu.
- Sélectionnez un mot aléatoire pour chaque partie.
- Vérifiez les lettres proposées par les clients et mettez à jour l'état du jeu.
- (Gestion multijoueur si le temps le permet)

Coté client :
- Créez une interface utilisateur (CLI) pour permettre aux joueurs de proposer des lettres.
- Envoyez les lettres proposées au serveur et affichez les réponses du serveur.
- Affichez visuellement l'état du pendu.

**Comment by HumairRomain** [comment]
repo : [https://github.com/HumairRomain/DAI-2023-24-Practical-work-2-TCP-Romain-Humair-and-Lucas-Lattion](url)

**Reply by lucaslattion** [reply]
Rendu: https://github.com/HumairRomain/DAI-2023-24-Practical-work-2-TCP-Romain-Humair-and-Lucas-Lattion/tree/13305f406ceee6776268e32f69eae1943c349084

Last Hash : 13305f406ceee6776268e32f69eae1943c349084
